.pre-commit-config.yaml.jinja

```
@@ -35,15 +35,15 @@ repos:
         name: isort except __init__.py
         args: [--settings, .]
         exclude: /__init__\.py$
-  # TODO Migrate to new prettier pre-commit repo after fixed:
-  # HACK https://github.com/prettier/pre-commit/pull/17
-  # HACK https://github.com/prettier/pre-commit/pull/18
-  - repo: https://github.com/prettier/prettier
-    rev: 2.1.2
+  - repo: https://github.com/prettier/pre-commit
+    rev: v2.1.2
     hooks:
       - id: prettier
         name: prettier + plugin-xml
         additional_dependencies:
+          # TODO: Remove this when https://github.com/prettier/pre-commit/pull/18
+          # or similar is fixed
```

I think I like this shorter:

```suggestion
          # HACK https://github.com/prettier/pre-commit/issues/16#issuecomment-713474520
```

Just a matter of taste.